mava/logger.py

```diff
@@ -102,19 +115,52 @@ def log(
                 f"Entropy {float(np.mean(entropy)):.3f}"
             )
             logger.console_logger.info(
-                f"{Fore.MAGENTA}{Style.BRIGHT}TRAINER: {log_string}{Style.RESET_ALL}"
+                f"{Fore.MAGENTA}{Style.BRIGHT}TRAINER{suffix}: {log_string}{Style.RESET_ALL}"
             )
         else:
             logger.console_logger.info(
-                f"{Fore.GREEN}{Style.BRIGHT}EVALUATOR: {log_string}{Style.RESET_ALL}"
+                f"{Fore.GREEN}{Style.BRIGHT}EVALUATOR{suffix}: {log_string}{Style.RESET_ALL}"
             )
 
         return float(np.mean(episodes_return))
 
-    return log
+    def save_json_by_seed() -> None:
```

Comment:
A thought on a possibly easier way to do this: instead of having this method, just creating different loggers for each seed. So now instead of a single logger you could have `Dict[int, Logger]` and then when logging you can keep it all the same just instead of `logger.xyz()` you could do `logger[seed].xyz()`.

I know this isn't a trivial change and so maybe we make an issue and address it next week?

Reply:
we wanted to do that but with creating ex.run in the main sacred won't allow having multiple loggers it will be just multiple references to the same instance :cry: